[2024-08-25T15:14:12.603+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-08-25T15:14:12.633+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: bigquery_dag.check_file manual__2024-08-25T15:14:06.906278+00:00 [queued]>
[2024-08-25T15:14:12.646+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: bigquery_dag.check_file manual__2024-08-25T15:14:06.906278+00:00 [queued]>
[2024-08-25T15:14:12.648+0000] {taskinstance.py:2856} INFO - Starting attempt 1 of 2
[2024-08-25T15:14:12.669+0000] {taskinstance.py:2879} INFO - Executing <Task(_PythonDecoratedOperator): check_file> on 2024-08-25 15:14:06.906278+00:00
[2024-08-25T15:14:12.678+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=7057) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-08-25T15:14:12.681+0000] {standard_task_runner.py:72} INFO - Started process 7061 to run task
[2024-08-25T15:14:12.681+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'bigquery_dag', 'check_file', 'manual__2024-08-25T15:14:06.906278+00:00', '--job-id', '143', '--raw', '--subdir', 'DAGS_FOLDER/gcp_integration_dag.py', '--cfg-path', '/tmp/tmp6zr3uv6u']
[2024-08-25T15:14:12.685+0000] {standard_task_runner.py:105} INFO - Job 143: Subtask check_file
[2024-08-25T15:14:12.759+0000] {task_command.py:467} INFO - Running <TaskInstance: bigquery_dag.check_file manual__2024-08-25T15:14:06.906278+00:00 [running]> on host ae952724060e
[2024-08-25T15:14:12.902+0000] {taskinstance.py:3122} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='sss_***' AIRFLOW_CTX_DAG_ID='bigquery_dag' AIRFLOW_CTX_TASK_ID='check_file' AIRFLOW_CTX_EXECUTION_DATE='2024-08-25T15:14:06.906278+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-08-25T15:14:06.906278+00:00'
[2024-08-25T15:14:12.904+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-08-25T15:14:12.910+0000] {logging_mixin.py:190} INFO - File does not exist
[2024-08-25T15:14:12.912+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-08-25T15:14:12.928+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-08-25T15:14:12.929+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=bigquery_dag, task_id=check_file, run_id=manual__2024-08-25T15:14:06.906278+00:00, execution_date=20240825T151406, start_date=20240825T151412, end_date=20240825T151412
[2024-08-25T15:14:12.979+0000] {local_task_job_runner.py:261} INFO - Task exited with return code 0
[2024-08-25T15:14:13.002+0000] {taskinstance.py:3891} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-08-25T15:14:13.007+0000] {local_task_job_runner.py:240} INFO - ::endgroup::
